src/sklearn_raster/datasets/_base.py

```diff
@@ -157,6 +192,38 @@ def load_swo_ecoplot(
     using all available Landsat imagery. Remote Sensing of Environment. 122:75–91.
     """
     X, y = sknnr.datasets.load_swo_ecoplot(return_X_y=True, as_frame=True)
+    variables = [
+        _VariableMeta("ANNPRE", "Annual precipitation", "ln mm"),
+        _VariableMeta("ANNTMP", "Mean annual temperature", "C"),
```

Comment:
I'm seeing the use of `degC` in the [CF conventions](https://cfconventions.org/cf-conventions/cf-conventions.html#temperature-units), but perhaps you have a difference convention that you're using.

Reply:
Good catch! I think I copied `C` from another dataset, but it isn't listed in the [UDUNITS database](https://docs.unidata.ucar.edu/udunits/current/#Database) which seems to be the official list.